src/sentry/static/sentry/app/views/releasesV2/list/index.tsx

```diff
@@ -135,5 +189,21 @@ class ReleasesList extends AsyncView<Props, State> {
   }
 }
 
+const StyledPageHeader = styled(PageHeader)`
+  flex-wrap: wrap;
+  margin-bottom: 0;
+  ${PageHeading} {
+    margin-bottom: ${space(2)};
+  }
+`;
+const SortAndFilterWrapper = styled('div')`
+  display: flex;
+  margin-bottom: ${space(2)};
+
+  & > *:not(:last-child) {
```

Comment:
What about using css grid instead of this complex selector

Reply:
❤️